lib/common/socket.c

```diff
@@ -1507,6 +1507,29 @@ void h2o_ssl_register_npn_protocols(SSL_CTX *ctx, const char *protocols)
 
 #endif
 
+int h2o_ssl_peer_verify_callback (int preok, X509_STORE_CTX *store_ctx)
+{
+    char data[256];
+
+    /* TODO: How much details are verified by default by openssl? */
```

Comment:
I would appreciate it if you could elaborate the concern here, otherwise it'd be hard to determine if the PR is ready.

Reply:
I think that by default the cert chain is verified. I hope that expiration date should be verified too, but I'm not sure. I don't think that we can verify CN or DN in this case, which is too much business dependency. I think that I need to remove the above if the above case is covered.

Reply:
Thanks, that makes sense.

As [stated in my other comment](https://github.com/h2o/h2o/pull/2621#discussion_r596461796), I tend to believe that verification of CN / DN is tied to the public root use-case, and that we can ignore that use-case for the sake of this PR.

Reply:
removed the logging.